Compile error in simple_text_input_protocol.zig

There is currently a compile error when using the `std.os.uefi.status.Status` struct:
```
lib/std/os/uefi/protocols/simple_text_input_protocol.zig:20:27: error: expected type 'std.os.uefi.status.Status', found 'usize'
        return self._reset(self, verify);
```

This can be fixed by replacing:

`_reset: fn (*const SimpleTextInputProtocol, bool) callconv(.C) usize,`

with

` _reset: fn (*const SimpleTextInputProtocol, bool) callconv(.C) Status,`

--- a/lib/std/os/uefi/protocols/simple_text_input_protocol.zig
+++ b/lib/std/os/uefi/protocols/simple_text_input_protocol.zig
@@ -11,7 +11,7 @@ const Status = uefi.Status;
 
 /// Character input devices, e.g. Keyboard
 pub const SimpleTextInputProtocol = extern struct {
-    _reset: fn (*const SimpleTextInputProtocol, bool) callconv(.C) usize,
+    _reset: fn (*const SimpleTextInputProtocol, bool) callconv(.C) Status,
     _read_key_stroke: fn (*const SimpleTextInputProtocol, *InputKey) callconv(.C) Status,
     wait_for_key: Event,
 

